### PR TITLE
[#321] remove word-break from notes column

### PR DESCRIPTION
**Pull request by paromita92**

### CODE REVIEW

File: app/assets/stylesheets/items.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+.item-index-status {
+  width: 7%;
+}
+
+.item-index-edit {
+  width: 5%;
+}
+
+.item-index-allocate{
```
Comment (nbanerjee) [inline app/assets/stylesheets/items.scss]: put space before ` {`

**Comment by ssinghi** [comment]
Why so many changes for what looks like a simple request?

/cc @nbanerjee

**Comment by paromita92** [comment]
@ssinghi  Sir, basically the table size is too wide so if i'm not not using different width's for each column the table column is breaking and so Rajiv da suggested me to give different width's  to the column's and delete some style's so that they don't override.

**Comment by ssinghi** [comment]
@paromita92 Please do only what is in the story. For that change a new story can be created and then worked upon.

**Pull request closed without merging**